pom.xml

```diff
@@ -31,6 +31,7 @@
         <module>infrastructures</module>
         <module>wsmaster/integration-tests</module>
         <module>assembly</module>
+        <module>tests/e2e</module>
```

Comment:
does it mean it will run everytime I build che-server?

Reply:
Every time you compile che-server, it should prepare tests. https://github.com/eclipse/che/pull/19524/files#diff-f8ed9675994a82a2c0a0d59b41c8395f50c117ccb62368aa8eae98b79ddedd66R53
Every time that `make-release.sh` is done (and so `maven deploy` is called) the test image should be pushed to npm registry. https://github.com/eclipse/che/pull/19524/files#diff-f8ed9675994a82a2c0a0d59b41c8395f50c117ccb62368aa8eae98b79ddedd66R92

I hope it make sense :) I've updated scripts today.

Reply:
thanks. how long does it takes to run? Can we exclude it in `fast` profile https://github.com/eclipse/che/blob/master/pom.xml#L1445 ?

Reply:
or better don't run it by default and include it in some dedicated maven profile? To me it feels like it should not be run on every build.